src/stories/textarea/TextArea.tsx

```diff
@@ -19,102 +20,72 @@ import { pget, theme } from '../theme';
  * ```
  * @constructor
  */
-export const TextArea: FC<InputProps & ComponentPropsWithoutRef<'textarea'>> = ({ status, disabled, ...props }) => {
-    const inputID = useId();
+export const TextArea: FC<InputProps & ComponentPropsWithoutRef<'textarea'>> = ({
+    label,
+    status,
+    type,
+    hideLabel,
+    description,
+    hideDescription,
+    startEnhancer,
+    endEnhancer,
+    disabled,
+    overrides,
+    ...props
+}) => {
+    const textareaID = useId();
     return (
-        // Disable a11y rules because the container doesn't need to be focusable for screen readers; the input itself should receive focus instead.
-        // The container is only made clickable for usability purposes.
-        // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
-        <div
-            {...props.overrides?.container}
-            className={clsx(
-                props.overrides?.container?.className,
-                styles.container,
-            )}
-            onClick={(e) => {
-                if (disabled) e.preventDefault();
-                if (typeof window !== 'undefined') {
-                    const input = document.getElementById(inputID);
-                    if (input && !disabled) {
-                        input.focus();
-                    }
-                }
+        <Field
```

Comment:
Not sure if this was intentional, to not use the Field component for `textarea`, couldn't find any difference so went ahead with it.